cmd/template/framework/files/makefile.tmpl

```diff
@@ -11,6 +11,14 @@ build:
 run:
 	@go run cmd/api/main.go
 
+# Create DB container
+docker-run:
+	@docker compose up -d
```

Comment:
this seems to be a plugin version for docker.
Lets go with ol' reliable:
`docker-compose up`

Reply:
I am learning now that `docker-compose` is V1 and was deprecated in June 2023.
This is a problem because I dont think it is backwards compatible from V2..

I think right now it is safer to keep it in V1 though... IDK